benches/containerd-shim-benchmarks/benches/wasi-demo-app-benchmarks.rs

```diff
@@ -0,0 +1,76 @@
+use std::process::Command;
+use std::time::{Duration, Instant};
+
+use criterion::{criterion_group, criterion_main, Criterion};
+
+fn run_container(runtime: &str, oci: bool) -> Duration {
+    let start = Instant::now();
+
+    let image_name = if oci {
+        "ghcr.io/containerd/runwasi/wasi-demo-oci:latest" // OCI artifact
+    } else {
+        "ghcr.io/containerd/runwasi/wasi-demo-app:latest"
+    };
+
+    let container_name = "testwasm";
+    let wasm_file = if oci {
+        "wasi-demo-oci.wasm"
+    } else {
+        "wasi-demo-app.wasm"
+    };
```

Comment:
It's probably cleaner and also extendable if you just pass in the `image_name` vs a bool flag

Reply:
Ah yes, thanks